### PR TITLE
Added a new VBD practical and updated solutions

### PR DESCRIPTION
**Pull request by arranhamlet**
Added solutions and data for practicals (Avian flu, Flu, HIV, VBD), split the VBD oncho practical into 2 apps (request from Jon).

Updated html in week 1 and week 2 to reflect this.

### CODE REVIEW

File: files/avian-influenza/Incidence_2.csv [hunk 1]
```diff
@@ -0,0 +1,53 @@
+MidTime,Count
```
Comment (richfitz) [inline files/avian-influenza/Incidence_2.csv]: can you please rename (and update links) to `incidence.csv` (note lower case) - I don't think that there is a version 1 of this file.

File: flu/flu_solution_1.R [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# initial conditions
```
Comment (richfitz) [inline flu/flu_solution_1.R]: elsewhere we don't have an underscore as `_1`, `_2`, etc.  These are also in the wrong directory - needs to be in `files/flu` not `flu`

File: flu/flu_solution_2.R [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# initial conditions
```
Comment (richfitz) [inline flu/flu_solution_2.R]: as for flu_solution_1.R

File: hiv/models/HIV_solution_1.R [hunk 4]
```diff
@@ -0,0 +1,42 @@
+##Differential equations
```
Comment (richfitz) [inline hiv/models/HIV_solution_1.R]: Move to `files/hiv/solution1.R`

File: hiv/models/HIV_solution_2.r [hunk 5]
```diff
@@ -0,0 +1,42 @@
+##Differential equations
```
Comment (richfitz) [inline hiv/models/HIV_solution_2.r]: `files/hiv/solution2.R` (note upper case R)

File: hiv/models/HIV_solution_3.R [hunk 6]
```diff
@@ -0,0 +1,87 @@
+##Differential equations
```
Comment (richfitz) [inline hiv/models/HIV_solution_3.R]: `files/hiv/solution3.R`

File: vbd/models/malaria_solution_1.R [hunk 7]
```diff
@@ -0,0 +1,30 @@
+initial(Xh) <- 99
```
Comment (richfitz) [inline vbd/models/malaria_solution_1.R]: `files/malaria/solution1.R`

File: vbd/models/malaria_solution_2.R [hunk 8]
```diff
@@ -0,0 +1,34 @@
+initial(Xh) = 99
```
Comment (richfitz) [inline vbd/models/malaria_solution_2.R]: as for solution 1

File: vbd/models/malaria_solution_3.R [hunk 9]
```diff
@@ -0,0 +1,36 @@
+initial(Xh) <- 99
```
Comment (richfitz) [inline vbd/models/malaria_solution_3.R]: as for solution 1

File: vbd/models/oncho_solution_1.R [hunk 10]
```diff
@@ -0,0 +1,42 @@
+initial(W) <- 10
```
Comment (richfitz) [inline vbd/models/oncho_solution_1.R]: as for solution 1

File: vbd/oncho_2/app.R [hunk 11]
```diff
@@ -0,0 +1 @@
+odin.ui:::odin_basic("model.R", show_table_summary = TRUE)
```
Comment (richfitz) [inline vbd/oncho_2/app.R]: should be `oncho2` not `oncho_2`, though this is not terribly clear. Perhaps there is a better naming system here, happy to discuss

File: week2.html [hunk 12]
```diff
@@ -49,7 +49,16 @@ <h3>Vector-borne diseases</h3>
 
         <ul>
           <li><a href="vbd/malaria">Malaria</a></li>
-          <li><a href="vbd/oncho">Oncho</a></li>
+          <li><a href="vbd/oncho">Oncho part 1</a></li>
+	  <li><a href="vbd/oncho_2">Oncho part 2</a></li>
```
Comment (richfitz) [inline week2.html]: If the first part is `oncho2`, then the first part must be `oncho1` - please update

**Review by richfitz — CHANGES_REQUESTED** [review]
bunch of renaming sorry

File: files/avian-influenza/Incidence_2.csv (hunk 1 above)
Comment (arranhamlet) [reply files/avian-influenza/Incidence_2.csv]: Updated, there is no version 1 - it was just called Incidence_v2.csv in the file supplied

File: flu/flu_solution_1.R (hunk 2 above)
Comment (arranhamlet) [reply flu/flu_solution_1.R]: Updated - as with other flu renaming/moving

**Comment by arranhamlet** [comment]
Changes requested updated in the new commits

**Review by richfitz — APPROVED** [review]

**Pull request merged**